src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js

```diff
@@ -22,15 +22,10 @@ const transform = ({
   fieldGovdeliveryIdEmerg,
   fieldGovdeliveryIdNews,
   fieldOperatingStatus,
-  fieldFacebook,
-  fieldFlickr,
-  fieldInstagram,
-  fieldTwitter,
   fieldNicknameForThisFacility,
   fieldRelatedLinks,
   fieldPressReleaseBlurb,
   fieldLinkFacilityEmergList,
-  fieldLeadership,
```

Comment:
Why is this being removed?

Reply:
Leadership field is no longer in use on health_care_region_page in cms - has been removed, @jhonnyoddball cc: @kevwalsh